test/Microsoft.Data.Analysis.Tests/DataFrameTests.IDataView.cs

```diff
@@ -224,5 +224,59 @@ public void TestIDataViewWithNulls()
             Assert.Equal("", preview.ColumnView[14].Values[5].ToString()); // null row
             Assert.Equal("foo", preview.ColumnView[14].Values[6].ToString());
         }
+
+        [Fact]
+        public void TestDataFrameFromIDataView()
+        {
+            DataFrame df = DataFrameTests.MakeDataFrameWithAllColumnTypes(10, withNulls: false);
+            df.Columns.Remove("Char"); // Because chars are returned as uint16 by IDataView, so end up comparing CharDataFrameColumn to UInt16DataFrameColumn and fail asserts
```

Comment:
> Because chars are returned as uint16 by IDataView

They are? where does this happen?

Reply:
Actually the comment is wrong. I meant to write `Because chars are returned as uint16 by DataViewSchema, ...`. See `PrimitiveDataFrameColumn.cs: GetDataViewType()`